backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java

```diff
@@ -1426,7 +1426,8 @@ public ExperimentRunPaginationDTO findExperimentRuns(
                 criteriaQuery,
                 experimentRunRoot,
                 authService,
-                roleService);
+                roleService,
+                ModelDBServiceResourceTypes.EXPERIMENT_RUN);
```

Comment:
Hm. What is this doing? We don't use experiment run as a resource type for most things.

Reply:
this is used for fuzzy search. the MDB call `getResources` with this type with workspace to getting entity owners for filter.

Reply:
As discussed, we cannot call getResources with those types. We should leave the logic intact.

Also, this PR is called "Fix find repositories logic using new visibility and workspace", so this change is clearly unrelated. Please don't mix multiple changes together in a single PR.

Reply:
@conradoverta i have did it on separate PR https://github.com/VertaAI/modeldb/pull/1839 but after merging PR in this that's why it is here.

Reply:
Removed unrelated changes to repository entity